Sources/Rendering/OpenGL/ImageResliceMapper/index.js

```diff
@@ -1133,17 +1138,16 @@ function vtkOpenGLImageResliceMapper(publicAPI, model) {
       resGeomString = resGeomString.concat(`Plane${slicePlane.getMTime()}`);
       if (image) {
         resGeomString = resGeomString.concat(`Image${image.getMTime()}`);
+        // Check to see if we can bypass oblique slicing related bounds computation
```

Comment:
I'm not sure I follow the reason for this change. Firstly, this function will not be called unless there is an input(https://github.com/Kitware/vtk-js/blob/a8b7a5cf99c546f28d92bad21ac71a3404c1bff8/Sources/Rendering/OpenGL/ImageResliceMapper/index.js#L121-L126). Secondly, even if execution gets here without an input, an xy plane should still be detected as orthonormal.